core/templates/services/UpgradedServices.ts

```diff
@@ -349,6 +349,8 @@ import { SkillDifficultyObjectFactory } from
   'domain/skill/SkillDifficultyObjectFactory';
 import { SkillMasteryBackendApiService } from
   'domain/skill/skill-mastery-backend-api.service';
+import { SkillMasteryObjectFactory } from
```

Comment:
This needs to be added in `oppia-angular-root.component.ts` as well. Please take a look at point 12 listed here: https://github.com/oppia/oppia/wiki/Angular-Migration#implementation-details-to-migrate-services.

cc/ @srijanreddy98

Reply:
Done.